ipyvolume/pylab.py

```diff
@@ -422,11 +431,17 @@ def quiver(x, y, z, u, v, w, size=default_size * 10, size_selected=default_size_
     """
     fig = gcf()
     _grow_limits(x, y, z)
-    scatter = ipv.Scatter(x=x, y=y, z=z, vx=u, vy=v, vz=w,
-                             color=color, size=size, color_selected=color_selected, size_selected=size_selected,
-                             geo=marker, **kwargs)
-    fig.scatters = fig.scatters + [scatter]
-    return scatter
+    if 'geo' in kwargs:
+        marker = kwargs['geo']
+        del kwargs['geo']
```

Comment:
Same here, please just use marker and forget about geo. I think maybe we should change Scatter.geo to Scatter.marker.

Reply:
if you changed Scatter.geo to Scatter.marker we can simplify some of this.

Reply:
:+1: